src/store/cache/user/graph/formatter.ts

```diff
@@ -35,7 +35,11 @@ export const formatUser = async (
       : { amount: new BN(0), lockupExpiryBlock: 0, target: '' }
   }
 
-  if (user.services.length === 0 && user.stakeAmount === '0')
+  if (
+    user.services.length === 0 &&
+    user.stakeAmount === '0' &&
+    user.claimableDelegationReceivedAmount === '0'
```

Comment:
just wondering what the deets are of this last user field?

Reply:
yeah, "claimable" in the graph schema means total minus all pending (pending because it's a 2-phase process to take tokens out of the system, typically there is a request, you wait a blockdiff and then you confirm)
Delegation is the amount of $AUDIO that a node operator receives
So, the claimable delegation received amount is the amount of tokens the node operator has received that is not in the current pending state